Topics/Development_Process/Docker.md

```diff
@@ -13,45 +13,201 @@
 
 This article will help readers understand what Docker is, why it is used and provide resources on how to start using it. Docker is used by developers for many reasons, however, the most common reasons are for building, deploying and sharing an application quickly. Docker packages your application into something that's called a [container](#docker-terminology-1). This [container](#docker-terminology-1) is OS-agnostic meaning that developers on Mac, Windows and Linux can share their code without any worry of conflicts. Here's [Amazon's Intro to Docker](https://aws.amazon.com/docker/#:~:text=Docker%20is%20a%20software%20platform,tools%2C%20code%2C%20and%20runtime.) if you want to learn more.
 
-----
-
 ## Installation
 
-To start using Docker you will have to download Docker Engine. This is automatically downloaded alongside Docker Desktop (A Graphical User Interface for Docker) which I **strongly** recommend for beginners.
-[Download Docker Desktop here](https://www.docker.com/get-started/)
-
+To start using Docker you will have to download Docker Engine. This is automatically downloaded alongside Docker Desktop (A Graphical User Interface for Docker) which I **strongly** recommend for Windows and macOS users.
+[Download Docker here](https://www.docker.com/get-started/)
 
 For detailed installation instructions based on specific operating systems click here: [Mac](https://docs.docker.com/desktop/install/mac-install/), [Windows](https://docs.docker.com/desktop/install/windows-install/), [Linux](https://docs.docker.com/desktop/install/linux-install/) 
 
-----
+## Creating Dockerfiles
 
-## Getting Started
-
-Once you've installed Docker, to see it in action you can follow any one of these quick tutorials:
+Once you've installed Docker, to see it in action you can follow any one of these quick tutorials on creating a Dockerfile that builds a docker image:
```

Comment:
I think overall everything looks really good! Just one thing is that you explain how Docker packages applications into a container, but it's kind of unclear if a Dockerfile is the file that contains the containerized application. Also what is a docker image, is it just a Dockerfile? I think people who are new to Docker might also be confused by this, so maybe you should further clarify what each term means.

Reply:
Fixed!